.github/workflows/release-otlp.yaml

```diff
@@ -0,0 +1,16 @@
+name: Release OTLP
```

Comment:
@jade-guiton-dd were you able to test this release in a Fork? We've had a lot of failed releases in the past because the job wasn't tested so now we require it.

Reply:
I haven't tested it yet. Is there something I need to setup on my fork for the CI to work, or do I just need to create a "vX" tag?

Reply:
You'll need a goreleaser-pro key.  @mx-psi or @jpkrohling can help if you don't already have access to one.

Reply:
I tested the release CI on my fork, and after a few adjustments (4c1f276) to point to my own Docker repo, and working around a problem which was currently affecting all release workflows (#676), it seems to work ([Github release](https://github.com/jade-guiton-dd/opentelemetry-collector-releases/releases/tag/v0.109.0-test3), [Docker repo](https://hub.docker.com/repository/docker/jadeguiton994/otelcol-release-tests/tags) if you want to check the result).